gramps_webapi/api/resources/emit.py

```diff
@@ -26,19 +26,22 @@ def __init__(self):
             getattr(lib, key) for key, value in inspect.getmembers(lib, inspect.isclass)
         ]
 
-    def response(self, payload: Any, args: Dict = {}) -> Response:
-        # pylint:disable=dangerous-default-value
+    def response(
+        self, status: int = 200, payload: Any = {}, args: Dict = {}
+    ) -> Response:
```

Comment:
*Never* use an empty dict as a default function argument. It is mutable and any modification that you make will persist. (See e.g. [here](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python)).

Use:

```python
def f(args: Optional[Dict] = None):
    args = args or {}
```

Reply:
I will fix this.